### DISCUSSION TITLE
How to deal with inital js in NextJs increase with NextUI?!

### DISCUSSION THREAD

**Discussion by J4v4Scr1pt** [heroui-inc/heroui#2128] [Help]
How do you people deal with the inital js size increases everytime you add a component?
I have only added a few and I'm already having red inital js in NextJS. 287kb..

**Comment by 0-don** [comment]
yes its a lot something needs to be changed maybe posibility to disable framer motion or something

**Reply by J4v4Scr1pt** [reply]
I don't think disable framer motion will be a good solution to this. 
My hope would be that NextJs could code-split out some 3rd party libs better.

But I managed to get it down to 187kb, by making sure we only used `'@nextui-org/COMPONENTNAME'` and not `'@nextui-org/react'`
I also added this to the next.config.
```
experimental: {
		optimizePackageImports: ['@nextui-org', '@nextui-org/react', 'framer-motion'],
	},
```

But it feels like these packages are already quite good optimized and this did not do much.
